### ISSUE TITLE
/latest/characters/%s/portrait/ return http:// links

### ISSUE THREAD

**Issue by biow0lf** [ccpgames/esi-issues#270]
Example response now:

```json
{
  "px64x64": "http://image.eveonline.com/Character/1337512245_64.jpg",
  "px128x128": "http://image.eveonline.com/Character/1337512245_128.jpg",
  "px256x256": "http://image.eveonline.com/Character/1337512245_256.jpg",
  "px512x512": "http://image.eveonline.com/Character/1337512245_512.jpg"
}
```

This is not right. If we ask API via http://, we should have http:// links. And if we ask via https://, we should have https:// links.

Right now, if we ask via http:// we get redirect to https. If we ask via https://, we get http:// links.

**Comment by biow0lf** [comment]
And the side effect, on iOS you will not need to disable `App Transport Security`. ATS require that app working via https and block any content which app try to load via http.

**Comment by CarbonAlabel** [comment]
This is already included in #131.

**Comment by a-tal** [comment]
closing this in favour of #131
